### ISSUE TITLE
[Feature request] Adaptation to the Whisper's JSON output 

### ISSUE THREAD

**Issue by sensboston** [EtienneAb3d/WhisperTimeSync#15]
Hello, is it possible to adapt your project to the Whisper's JSON output? I'm working on karaoke program for Windows, and need all words in the lyrics to be timestamped.
I'll be glad to issue a PR for this feature but unfortunately I'm not proficient in Python programming (mostly use a C# & C++).

**Comment by EtienneAb3d** [comment]
@sensboston 

I will have a look at it as soon as possible.
To be sure of what you expect, can you provide me with an example (JSON+TXT)?

WhisperTimeSync is not written in Python but in Java.
;-)

**Comment by sensboston** [comment]
Here we go: [samples.zip](https://github.com/EtienneAb3d/WhisperTimeSync/files/14387291/samples.zip)

**Comment by EtienneAb3d** [comment]
@sensboston 
Hmmm... The problem I see with this JSON format is that each word has a **mandatory** description including its timestamp. It will be very hard to decide what to do with not-matching words.
🤔

**Comment by sensboston** [comment]
Yeah, it's an issue, agree. But I haven't looked to your (or Java code you've ported) implementation (yet). Theoretically it's possible, even without involving AI - for example by using "soundex" algorithm. I thought about this but initially wanna check if someone already done this.

**Comment by sensboston** [comment]
No budget at all, I do development just for fun, will publish open source here when it done.

**Comment by sensboston** [comment]
Any progress?

**Comment by EtienneAb3d** [comment]
@sensboston 
I understood you were working on the subject on your side.
On my side, without a budget, I have to find/allocate on my free time.
